package/MDAnalysis/coordinates/chain.py

```diff
@@ -211,7 +212,19 @@ def convert_pos_to_native(self, x):
         return self.active_reader.convert_pos_to_native(x)
 
     def copy(self):
-        raise NotImplementedError("Copy not implemented for ChainReader")
+        #raise NotImplementedError("Copy not implemented for ChainReader")
+        new = self.__class__(self.filenames)
```

Comment:
If you're passing in the filenames here, you probably don't have to then copy the Readers.  I'd say you can do one or the other, so maybe see if you can get away with just passing filenames.

You might have to `copy.copy(self.filenames)` else they both share a reference to the same Python list